hazelcast/src/test/java/com/hazelcast/client/test/CustomCredentials.java

```diff
@@ -23,14 +23,16 @@
  */
 public class CustomCredentials extends UsernamePasswordCredentials {
 
+    public static final int CLASS_ID = 14;
```

Comment:
is this correct? I see that `TaskWithUnserializableResponse.CLASS_ID` as 14

Reply:
Good catch.